tests/test_properties.cc

```diff
@@ -27,8 +27,11 @@
  * Boston, MA 02111-1307, USA.
  */
 
+#include <boost/test/unit_test.hpp>
 #include "test_properties.hh"
 
+constexpr double TOLERANCE = 1e-10;
```

Comment:
This tolerance is different than the one in the file above. I think it was 1e-14. Should they be the same?
I mean it is so small that we probably do not bother anyways...

Reply:
the previous global tol was set to 1e-12, and this file used it as tol * 100, making it 1e-10

In other files, tol was used as tol/100 making it 1e-14, or directly, so I followed that. 

I guess we could try to unify the tolerance, but this should be part of a separated PR.